spec/models/manageiq/providers/amazon/cloud_manager/event_target_parser_spec.rb

```diff
@@ -1669,6 +1669,6 @@ def response(path)
   def create_ems_event(path)
     event = ManageIQ::Providers::Amazon::CloudManager::EventCatcher::Stream.new(double).send(:parse_event, response(path))
     event_hash = ManageIQ::Providers::Amazon::CloudManager::EventParser.event_to_hash(event, @ems.id)
-    EmsEvent.add(@ems.id, event_hash)
+    EmsEvent.add(@ems.id, event_hash.deep_symbolize_keys)
```

Comment:
This simulates how the event_hash is processed by the event_handler: https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_event_handler/runner.rb#L3